wit/wasi-nn.wit

```diff
@@ -1,4 +1,4 @@
-package wasi:nn;
+package wasi:nn@0.2.0;
```

Comment:
```suggestion
package wasi:nn@0.2.0-rc-2024-06-25;
```

Reply:
For reference, this is what other proposals did prior to stabilization, e.g., [here](https://github.com/WebAssembly/wasi-filesystem/commit/76eb8862fa43c6267adb907f13bcbf5a16236362).